src/set_theory/ordinal/arithmetic.lean

```diff
@@ -586,9 +587,28 @@ instance : monoid ordinal.{u} :=
     ⟨⟨prod_punit _, λ a b, by rcases a with ⟨a, ⟨⟨⟩⟩⟩; rcases b with ⟨b, ⟨⟨⟩⟩⟩;
     simp only [prod.lex_def, empty_relation, and_false, or_false]; refl⟩⟩ }
 
-@[simp] theorem type_prod_lex {α β : Type u} (r : α → α → Prop) (s : β → β → Prop)
+theorem type_prod_lex {α β : Type u} (r : α → α → Prop) (s : β → β → Prop)
   [is_well_order α r] [is_well_order β s] : type (prod.lex s r) = type r * type s := rfl
 
+theorem type_lt_prod_lex {α β : Type u} [has_lt α] [has_lt β]
+  [is_well_order α (<)] [is_well_order β (<)] :
+  @type (β ×ₗ α) (<) _ = @type α (<) _ * @type β (<) _ :=
+type_prod_lex _ _
+
+@[simp] theorem type_prod_lex_lift {α : Type u} {β : Type v} (r : α → α → Prop) (s : β → β → Prop)
```

Comment:
Maybe
```suggestion
@[simp] theorem type_prod_lex_eq_lift {α : Type u} {β : Type v} (r : α → α → Prop) (s : β → β → Prop)
```
since the `lift` is on the RHS

Reply:
I think I'll rename `ordinal.type_prod_lex` to `ordinal.prod_def` and `ordinal.type_prod_lex_lift` to `ordinal.type_prod_lex`. This is somewhat analogous to what we do with `cardinal.prod_def` and `cardinal.mk_prod`.

Reply:
> This is analogous to what we do with `cardinal.mul_def` and `cardinal.mk_prod`.

`cardinal.mul_def` has the `*` on the LHS; you should do the same thing with `ordinal.mul_def`, as I mention above.